app/services/gplaces.js

```diff
@@ -17,5 +17,32 @@ module.exports = function() {
           });
     });
   };
+  this.getPlaceByText = (query, optionalParameters)=>{
+    return new Promise((resolve, reject)=>{
+      let url;
+      if(optionalParameters==undefined)
+      {
+        url="https://maps.googleapis.com/maps/api/place/textsearch/json?query="
```

Comment:
Space zwischen Text und `=`

Reply:
So ist das nicht in der lint config definiert, das ist so in Ordnung